usaspending_api/common/api_request_utils.py

```diff
@@ -289,35 +290,71 @@ class AutoCompleteHandler():
     @staticmethod
     # Data set to be searched for the value, and which fields to look in
     # Mode is either "contains" or "startswith"
-    def get_values_and_counts(data_set, fields, value, mode="contains"):
+    def get_values_and_counts(data_set, filter_matched_ids, pk_name):
         value_dict = {}
         count_dict = {}
 
+        for field in filter_matched_ids.keys():
+            q_args = {}
```

Comment:
Looking at it more, I'm pretty sure the intent is for `q_args` to only have one key each time through, so this isn't a bug, just an optional code style suggestion.